osmosis-1/osmosis-1.assetlist.json

```
@@ -562,6 +562,35 @@
         "svg": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/btsg.svg"
       },
       "coingecko_id": "bitsong"
+    },
+    {
+      "description": "Panacea is a blockchain which is the key infrastructure for our services to reinvent the healthcare experience",
+      "denom_units": [
+        {
+          "denom": "ibc/0927C519BFAF5F57DAF6357CFE25CCC4C8956245F9C8B5A1F161CFCFB8A51851",
+          "exponent": 0,
+          "aliases": [
+            "umed"
+          ]
+        },
+        {
+          "denom": "med",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/0927C519BFAF5F57DAF6357CFE25CCC4C8956245F9C8B5A1F161CFCFB8A51851",
+      "name": "Panacea",
```

```suggestion
      "name": "MediBloc",
```
Ah. Let's go with the name `MediBloc` which would be more clear for users who will use DEXs (since we are listed as MediBloc on CEXs).

---

Ah.. Let's go with the `Panacea` as you wrote first because MediBloc might launch multiple chains in the future.